src/components/Property.tsx

```diff
@@ -27,18 +27,21 @@ const Property = ({ listing }: { listing: PropertyType }) => {
   //needed use effect to access promise from the convertaddress function
   // (we should do this before it goes into the database and get the data
   //from the property object instead of using a useeffect for this.)
-  const fullAddress = `${listing.address1}, ${
-    listing.address2 && listing.address2 + ', '
-  }${listing.city}, ${listing.postcode}`;
+  const fullAddress = `${listing.address1}, ${!listing.address2 ? '' : listing.address2 && listing.address2 + ', '
+  }${listing.postcode}, ${listing.city}, UK`;
   useEffect(() => {
-    convertAddress(fullAddress)
-      .then((location) => {
-        setCenter(location);
-        setMarkers([location]);
-        setLoading(false);
-      })
-      .catch((error) => console.error('something has gone wrong'));
-  }, [fullAddress]);
+    setCenter(center);
+    setMarkers([center]);
+    setLoading(false);
+    // convertAddress(fullAddress)
+    //   .then((location) => {
+    //     console.log('🎯',location)
+    //     setCenter(location);
+    //     setMarkers([location]);
+    //     setLoading(false);
+    //   })
+    //   .catch((error) => console.error('something has gone wrong'));
```

Comment:
can we just get rid of this chunk of ghost code before merging? I think we're steadily accumulating quite a bit of comments and throwaway stuff

Reply:
yep, I think I had left it in in case we wanted to chat it through.

Reply:
Thanks. I think as long as the changes made work and pass the tests we can all be ruthless with the code base and delete old versions 😅 (maybe excepting complex functionality apt to affect other aspects of the app, but this wasn't gonna be the case